posthog/temporal/batch_exports/destination_tests.py

```diff
@@ -805,7 +805,102 @@ async def _run_step(self) -> DestinationTestStepResult:
                 else:
                     return DestinationTestStepResult(
                         status=Status.FAILED,
-                        message=f"Could not use Snowflake database '{self.warehouse}'. Received error: {err}.",
+                        message=f"Could not use Snowflake database '{self.database}'. Received error: {err}.",
```

Comment:
Also a small fix here as this message should contain the database not the warehouse.